src/services/API/sqs.js

```diff
@@ -20,7 +20,7 @@ const sqsClient = new SQSClient({
 * Everything, except the userIDList, is extracted from the auth layer URL
 */
 
-export async function sendSQSMessage(purpose, purposeParams, redirectTo, redirectID, userIDList, authType) {
+export async function sendSQSMessage(purpose, purposeParams, redirectTo, redirectID, userIDList, authType, programName) {
```

Comment:
as mentioned in a previous comment, programName feels like it should be a part of purposeParams